libs/ui/src/lib/datepicker/datepicker-calendar.component.ts

```diff
@@ -0,0 +1,150 @@
+import {
```

Comment:
Should we have a story for `DatepickerCalendarComponent`, too? Is it ready to be used outside of the lib?

Reply:
no, i only exported it because the compiler complained about it